scene/resources/compressed_texture.cpp

```diff
@@ -807,11 +807,10 @@ RID CompressedTextureLayered::get_rid() const {
 }
 
 Ref<Image> CompressedTextureLayered::get_layer_data(int p_layer) const {
-	if (texture.is_valid()) {
```

Comment:
Since this was handled without an error previously, I assume it might have been intentional, which would mean that this function can be called at times where a texture wasn't defined, without being an error. Did you evaluate whether that's the case?

Reply:
> which would mean that this function can be called at times where a texture wasn't defined, without being an error.

Yes it previously works this way and is consistent with all other texture's similar implentations —— return am empty Image without being an error. I will change my code.